oss-internship-2020/curl/examples/example1.cc

```diff
@@ -0,0 +1,78 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include <libgen.h>
+#include <syscall.h>
+#include <iostream>
+#include "curl_sapi.sapi.h"
+#include "sandboxed_api/util/flag.h"
+
+/*
```

Comment:
Please do not include commented code. If you want to, you can add a detailed comment in `main()`, explaining that this example replicates what this code block does.